README.md

```diff
@@ -10,6 +10,8 @@ Canadarm makes JavaScript logging easy and seamless, helping you figure out what
 Named for the [Canadarm](http://en.wikipedia.org/wiki/Canadarm) we hope this too helps you piece together
 log information to figure out what went wrong with your scripts.
 
+If you have any ideas or questions reach out to us on our [Google group][google-group-url].
```

Comment:
I'm not going to worry too much about it. In what I'm reading the "If you have any ideas or questions" is not an independent clause so the comma is not needed. Nick Overfield may have thoughts. 

Thanks for reviewing, I'll give it a merge.